reference/conanfile/methods.rst

```diff
@@ -1445,3 +1445,93 @@ multiple recipes.
             # customize toolchain "tc"
             tc.generate()
             # Or provide your own custom logic
+
+
+layout()
+--------
+
+.. warning::
+
+    This is an **experimental** feature subject to breaking changes in future releases.
+
+Available since: `1.37.0 <https://github.com/conan-io/conan/releases/tag/1.37.0>`_
+
+Read about the feature :ref:`here<package_layout>`.
+
+In the layout() method you can adjust ``self.folders``, ``self.cpp`` and ``self.patterns``.
+
+self.folders
+++++++++++++
+
+- **self.folders.source** (Defaulted to ""): Specifies a subfolder where the sources are. The ``self.source_folder`` attribute and
```

Comment:
If this is a copy of the other section, maybe leave this here, and simplify/reduce a little bit the other section, and use cross-reference?